libcrux-ml-kem/src/serialize.rs

```diff
@@ -225,13 +378,16 @@ pub(super) fn compress_then_serialize_ring_element_v<
 }
 
 #[inline(always)]
+#[hax_lib::requires(
+    serialized.len() == 320
+)]
 fn deserialize_then_decompress_10<Vector: Operations>(
     serialized: &[u8],
 ) -> PolynomialRingElement<Vector> {
-    hax_debug_assert!(serialized.len() == (COEFFICIENTS_IN_RING_ELEMENT * 10) / 8);
-
+    hax_lib::fstar!("assert (v (($COEFFICIENTS_IN_RING_ELEMENT *! sz 10) /! sz 8) == 320)");
     let mut re = PolynomialRingElement::<Vector>::ZERO();
 
+    let _coefficients_length = re.coefficients.len();
```

Comment:
I might be missing something, but I don't see this used. Or is this something cloop-specific?

Reply:
Good catch, removing.